api/types/filters/parse_test.go

```diff
@@ -397,3 +398,97 @@ func TestClone(t *testing.T) {
 	f2.Add("baz", "qux")
 	assert.Check(t, is.Len(f.Get("baz"), 0))
 }
+
+func TestParseBoolIfPresent(t *testing.T) {
+
+	for _, tC := range []struct {
```

Comment:
Can you give all the test cases a name and run them in a subtest? It helps when something is failing what test case it's failing on.

Reply:
Will do.